src/test/java/tests/boardTests/listTests/cardTests/DeleteChecklistTest.java

```diff
@@ -0,0 +1,67 @@
+package tests.boardTests.listTests.cardTests;
+
+import api.clients.ApiCardClient;
+import api.clients.ApiChecklistClient;
+import api.clients.ApiListClient;
+import api.pojo.requests.CardBuilder;
+import api.pojo.requests.ChecklistBuilder;
+import api.pojo.requests.ListBuilder;
+import jdk.jfr.Description;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import tests.TestInit;
+import ui.pages.BoardPage;
+import ui.pages.TrelloHomePage;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.refresh;
+
+public class DeleteChecklistTest extends TestInit {
```

Comment:
I guess you had to make this test in CreateChecklistTest

Reply:
CreateChecklistTest is one testm DeleteChecklistTest is another one. I think we can leave it at that.